test/e2e/nbthread_test.go

```diff
@@ -0,0 +1,87 @@
+// +build e2e
+
+package e2e
+
+import (
+	"context"
+	"strconv"
+	"testing"
+	"time"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+
+	"github.com/openshift/cluster-ingress-operator/pkg/operator/controller"
+	"github.com/openshift/cluster-ingress-operator/pkg/operator/controller/ingress"
```

Comment:
nit: name this import `ingresscontroller`

Reply:
done